suprise/main.py

```diff
@@ -0,0 +1,8 @@
+
+#a decorator for any function
+#it will make the function prit suprise! instead of its original functionality
+def print_surprise(function):
+    def inside(*args):
+        print("surprise!")
+    return inside
+
```

Comment:
File has no top level environment of the program  
https://docs.python.org/3/library/__main__.html